.github/workflows/playwright.yml

```diff
@@ -6,7 +6,7 @@ on:
     branches: [develop, release-al2, master]
 jobs:
   test:
-    timeout-minutes: 60
+    timeout-minutes: 40
```

Comment:
Just for clarity sake, this first time-out is for the entire test? 

Then the subsequent timeout is for each step right? But the total sum of the subsequent timeout is 30 vs the parent 40, then do we even need this line? 😅 Or are we factoring in the `build` time?

Reply:
Yes, it includes build time. And I saw a successful run with 32min, that's why it's 40min here.

Reply:
this is one is for the whole job